### PR TITLE
Update to Drupal 9. DDSDK-499 / DDSDK-500

### PR DESCRIPTION
**Pull request by rasben**

### CODE REVIEW

**Comment by rasben** [comment]
**⚠️ Search API index  needs to be re-indexed when this is deployed! ⚠️**

**Comment by rasben** [comment]
The visual errors are a bug - its not actually happening.

**Comment by rasben** [comment]
This has been tested 👍 by Anders & Co.

File: configuration/drupal/taxonomy.vocabulary.activity_type.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 uuid: 961b299c-f043-4010-a71e-7509524e6fc6
-langcode: en
+langcode: da
 status: true
 dependencies: {  }
 name: 'Activity type'
```
Comment (kasperg) [inline configuration/drupal/taxonomy.vocabulary.activity_type.yml:5]: Is this change intentional? It seems as if we change the language but do not change other language-related aspects. E.g. the language is now Danish but the name remains "Activity type".

Comment (rasben) [reply configuration/drupal/taxonomy.vocabulary.activity_type.yml]: It's intentional that i'm changing it to langcode DA, but I dont know if i can call the missing label "intentional"

I mean, on the current site, there is no translation of 'activity type', so it's not like I've broken anything that was working already...

Regardless - I've translated it now:
https://github.com/reload/ddsdk2016/compare/309b4881ca8be8b83f5b0957d3ea1a049338f413..197a77d050bfd5f8c7c172f7a97b27c36182f24e

Comment (rasben) [reply configuration/drupal/taxonomy.vocabulary.activity_type.yml]: Oh - and the label is only used for the editors - not displayed in frontend regardless :)

**Review by kasperg — APPROVED** [review]
:+1: LGTM
